x/auction/types/genesis_test.go

```diff
@@ -0,0 +1,75 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/Stride-Labs/stride/v5/x/auction/types"
+)
+
+func TestGenesisState_Validate(t *testing.T) {
+    for _, tc := range []struct {
+    		desc          string
+    		genState      *types.GenesisState
+    		valid bool
+    } {
+        {
+            desc:     "default is valid",
+            genState: types.DefaultGenesis(),
+            valid:    true,
+        },
+        {
+            desc:     "valid genesis state",
+            genState: &types.GenesisState{
+            	
+                AuctionPoolList: []types.AuctionPool{
+	{
```

Comment:
This looks a bit messy. Can you please clean up this file?
Not only this one but please also check all other files you have written again and clean this kind of styling issues.